frontend/src/components/JiraDashboard/components/jira/Jira.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "react-notifications/lib/notifications.css";
+import { NotificationContainer } from "react-notifications";
+
+import useAuthorize from "../jiraAuthorization/useAuthorize";
+import JiraAuth from "../jiraAuthorization/JiraAuth";
+import Notification from "../notification/Notification";
+import Widgetjira from "../WidgetJira";
+import Loader from "../../../../globalComponents/Loader/Tombstone";
+
+const Jira = () => {
+    const { doneAuthentication, loading } = useAuthorize();
+    console.log("Jiraa", doneAuthentication);
+    return (
```

Comment:
don't use multiple ternary experessions. in case you run into more than 2 conditions/cases break it down into switch. or return early for loading case